src/app/static/src/app/components/Stepper.vue

```diff
@@ -146,6 +146,11 @@
             VersionStatus
         },
         watch: {
+            complete: function (){
+                if (this.activeStep === 4){
```

Comment:
Initially I had the if statement be slightly stricter for this, requiring calibrate options step to be enabled. 
`if (this.activeStep === 4 && this.isEnabled(5)){`
However, I was not able to get the tests for this to pass as the criteria for getting the calibrate to be enabled were very strict and I failed at mocking them up. Should everything else in the app be working correctly, this shouldn't be an issue as you would normally expect the calibrate step to be enabled once the model run completed. However, ideally the stricter criteria would be maintained so I'd welcome any feedback about the tests.

Reply:
I think the problem with isEnabled is that it requires all previous steps to be complete, but in the case of your test, the earlier steps like baseline are not complete, as defined in the stepper getter 'complete' (which farms out module level logic to getters in baseline etc). You could bodge up baseline state etc which does satisfy the getters that it's complete, or you could just mock the stepper getters to give back the complete values you want. This would mean breaking open createSut though - let's talk about this if not clear.